packages/app/src/components/Live/LiveChat.tsx

```diff
@@ -33,24 +38,34 @@ export const LiveChat = ({
 }: Props): ReactElement => {
     const [inputVisible, setInputVisible] = useState(false);
     const [animation, setAnimation] = useState(false);
-    const [moveAnimation] = useState(() => new Animated.Value(0));
+    const moveAnimation = useSharedValue(0);
 
     const handlePress = () => {
         setAnimation(!animation);
         if (!inputVisible) {
-            moveUp(-55);
             setInputVisible(true);
+            moveAnimation.value = -55;
```

Comment:
Does this take percentages? It might be a more scalable solution to use % of screen here

Reply:
It depends on the height of the input container, so it's just been updated it to use that.